src/config.rs

```diff
@@ -1666,6 +1674,190 @@ pub fn persist_critical_config(config: &TiKvConfig) -> Result<(), String> {
     Ok(())
 }
 
+fn to_config_entry(change: ConfigChange) -> CfgResult<Vec<configpb::ConfigEntry>> {
+    fn helper(prefix: String, change: ConfigChange) -> CfgResult<Vec<configpb::ConfigEntry>> {
+        let mut entries = Vec::with_capacity(change.len());
+        for (mut name, value) in change {
+            if name == "raft_store" {
```

Comment:
Does it cause by `serde(rename=...)`? If so, you can use rename the field to align with the name in toml file.

Reply:
Yes it is.